src/layouts/Header.tsx

```diff
@@ -7,7 +7,6 @@ const Header = () => {
       <Head>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="description" content="CourtCanva" />
-        <link rel="icon" href="/favicon.ico" />
```

Comment:
That's a great approach, but I feel it's unrelated to the content of this card. Perhaps we could create a separate card later to address these irrelevant pieces of code.